src/sys.zig

```diff
@@ -100,6 +105,7 @@ pub const Tag = enum(u8) {
     utimes,
     write,
     getcwd,
+    getenv,
     chdir,
```

Comment:
☝️ Added `getenv` to `Syscall.Tag` enum for some clarity if needed in the `Syscall.Error`.